lib/api/users/user.ts

```diff
@@ -0,0 +1,19 @@
+import { db} from "../../db";
+
+export function register(building: number, number: number, name: string, email: string) {
+
+    const user_statement = db.prepare("INSERT INTO users (name, email) VALUES(?, ?)").run(name, email)
+    const statement = db.prepare("INSERT INTO registrations (building_id, num, reported_at, user_id) VALUES (?, ?, ?, ?)")
+
+    const info = statement.run(building, number, Date.now(), user_statement.lastInsertRowid)
+    return info.lastInsertRowid
```

Comment:
I'd recommend using a different return value - as documented, `lastInsertRowid` is to be ignored when no change occurs - as such, this doesn't provide information about whether the operation was successful